.maestro/run_ui_tests.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/zsh
+
+### Run UI tests
+
+source .maestro/common.sh
+
+## Constants
+
+run_log="$derived_data_path/run_log.txt"
+app_bundle="com.duckduckgo.mobile.ios"
+
+## Functions
+
+fail() {
+    echo "‼️ $1"
+    exit 1
+}
+
+check_is_root() {
```

Comment:
This is technically duplicated now... even though, considering the issue in the source command, it's actually the definition that it's used when the script is invoked from the wrong folder.